forge-gui/res/adventure/Shandalar/custom_cards/nahiris_boss_effect.txt

```diff
@@ -0,0 +1,10 @@
+Name:Nahiri Boss Effect
+ManaCost:no cost
+Types:Enchantment
+S:Mode$ ReduceCost | ValidCard$ Card | ValidSpell$ Activated.Equip | Activator$ You | Amount$ 1 | EffectZone$ Command | Description$ Equip costs you pay cost {1} less.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Command | Secondary$ True | Execute $ TrigConjure | TriggerDescription$ At the beginning of your upkeep or when an equipped creature you control attacks, conjure a card from Nahiri's Armory into your hand
```

Comment:
Similar to the comment on Nahiri's Armory, this one is a little weirder still - Either the card should be named Nahiri's Armory and have "'s spellbook" added into this line, or reword it to "Nahiri Boss Effect's spellbook"